### ISSUE TITLE
The browser library should automatically detect the current page for the redirect URL

### ISSUE THREAD

**Issue by jaxoncreed** [inrupt/solid-client-authn-js#421]
In solid-auth-client, providing a redirect url was optional because in the browser, it would automatically detect the current url and use that. The same should apply for solid-client-authn-js-browser.

**Comment by pmcb55** [comment]
Yeah sure, sounds like a simple suggestion. I've created a ticket in our backlog - but if you (or anyone else) get around to doing it, just reference the PR here!
